examples/marketdataV2.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * This examples shows how to use tha alpaca data v2 rest functions during some
+ * simple examples. Don't forget to install the dependencies before running it.
+ * The results are logged to the console in a readable tabled form.
+ * This file was written in Typescript, to run it use eg. the ts-node command.
+ */
+
+import Alpaca from "@alpacahq/alpaca-trade-api";
+import * as _ from "lodash";
+import moment from "moment";
+
+class Example {
+  async run() {
+    // Create your alpaca object by setting your credentials
```

Comment:
I don't think this example should be a class. In reality this should probably just be an async IIFE  or just an async function named example :thinking: